kpet/__init__.py

```diff
@@ -98,6 +98,18 @@ def build_run_command(cmds_parser, common_parser):
         default=[],
         help='List of mbox URLs/paths comprising the patch series'
     )
+    print_test_cases_parser = action_subparser.add_parser(
+        "print_test_cases",
+        help="Print test cases according to the patches",
```

Comment:
I would rather say something like "applicable to the patches". Because patches are not saying directly what tests to run.

Reply:
Done

Reply:
Ups :), fixed